buildSrc/build.gradle.kts

```diff
@@ -5,14 +5,6 @@ plugins {
   id("com.diffplug.spotless") version "5.13.0"
 }
 
-spotless {
```

Comment:
`buildSrc` is esssentially a completely independent project, so it doesn't actually get to use anything defined here. It's why we even need to keep the spotless versions in sync between the `plugins` block above (used by buildSrc) and the `dependencies` block below (used by the main build) :(